x-pack/plugins/upgrade_assistant/mappings.json

```diff
@@ -0,0 +1,33 @@
+{
+  "upgrade-assistant-reindex-operation": {
+    "properties": {
+      "indexName": {
+        "type": "text"
```

Comment:
Are we doing any full-text searching on these? If not we can change them to all be `keyword`

Reply:
No we're not, good idea.

Reply:
Changed a dynamic mapping with the only defined fields as the two we every query on: indexName and status.